lib/api_me/sorting.rb

```diff
@@ -19,9 +19,7 @@ def sort_meta
       return {} unless sorting?
       {
         criteria: sort_meta_criteria,
-        reverse: sort_reverse,
-        record_count: scope.size,
```

Comment:
Interesting that these where here. I don't suspect they were used.

Reply:
`record_count` and `total_records` are for pagination and not sorting. I'm not sure why they were there.